projects/aws/eks-a-admin-image/README.md

```diff
@@ -1,19 +1,19 @@
-# Snow EKS-A Admin image
-This project builds an VM image compatible with Snow devices containing all necessary artifacts to manage EKS-A clusters.
+# EKS-A Admin image
+This project builds an EKS-A Admin VM image containing all necessary artifacts to manage EKS-A clusters. 
 
 It first uses `packer` to build an AMI and then it exports it to RAW format using `aws ec2 export-image`.
```

Comment:
This only applies to the snow one, I think

Can we add a section talking about the two different image types we support and how they work? It would also be super useful a small section about making provisioners exclusive to one or the other image type.

Reply:
Yes I agree this will be useful. I will update.